spec/unit/matrix-client.spec.ts

```diff
@@ -342,6 +339,11 @@ describe("MatrixClient", function () {
         store.getClientOptions = jest.fn().mockReturnValue(Promise.resolve(null));
         store.storeClientOptions = jest.fn().mockReturnValue(Promise.resolve(null));
         store.isNewlyCreated = jest.fn().mockReturnValue(Promise.resolve(true));
+
+        unstableFeatures = {
+            "org.matrix.msc3440.stable": true,
+        };
```

Comment:
```suggestion
        unstableFeatures["org.matrix.msc3440.stable"] = true;
```

(though I'm not sure why it is being moved at all)

Reply:
This sets `unstableFeatures` to a defined state before each test. Another test sets an additional property on it: https://github.com/matrix-org/matrix-js-sdk/blob/deb908947b06047dc93bd5f82906e63133f9d3fd/spec/unit/matrix-client.spec.ts#L1400

Reply:
→ will add a comment

Reply:
oh, I see.

Reply:
oh, I see.